disttask/framework/dispatcher/BUILD.bazel

```diff
@@ -16,6 +17,7 @@ go_library(
         "//util/logutil",
         "//util/syncutil",
         "@com_github_pingcap_errors//:errors",
+        "@com_github_stretchr_testify//mock",
```

Comment:
I suggest we can add ```race = "on",``` to ```go_test``` for finding the data race. You can decide whether to open it according to the situation